src/includes/javascript-crons-upsert.mdx

```diff
@@ -0,0 +1,124 @@
+You can create and update your Monitors programmatically with code rather than [creating and configuring them in Sentry.io](https://sentry.io/crons/create/).
+
+Using `Sentry.withMonitor()`, you can pass a third parameter with your monitor configuration:
+
+```javascript
+Sentry.withMonitor(
+  "<monitor-slug>",
+  () => {
+    // Execute your scheduled task here...
+  },
+  {
+    schedule: {
+      type: "crontab",
+      value: "* * * * *",
+    },
+    checkinMargin: 2, // In minutes. Optional.
+    maxRuntime: 10, // In minutes. Optional.
+    timezone: "America/Los_Angeles", // Optional.
+  }
+);
+```
+
+Using `Sentry.captureCheckIn()`, you can pass a second parameter with your monitor configuration:
+
+```javascript
+const monitorConfig = {
+  schedule: {
+    type: "crontab",
+    value: "* * * * *",
+  },
+  checkinMargin: 2, // In minutes. Optional.
+  maxRuntime: 10, // In minutes. Optional.
+  timezone: "America/Los_Angeles", // Optional.
+};
+
+// 🟡 Notify Sentry your job is running:
+const checkInId = Sentry.captureCheckIn(
+  {
+    monitorSlug: "<monitor-slug>",
+    status: "in_progress",
+  },
+  monitorConfig
+);
+
+// Execute your scheduled task here...
+
+// 🟢 Notify Sentry your job has completed successfully:
+Sentry.captureCheckIn(
+  {
+    // Make sure this variable is named `checkInId`
+    checkInId,
+    monitorSlug: "<monitor-slug>",
+    status: "ok",
+  },
+  monitorConfig
+);
+```
+
+```typescript
+import { MonitorConfig } from "@sentry/types";
+
+const monitorConfig: MonitorConfig = {
+  schedule: {
+    type: "crontab",
+    value: "* * * * *",
+  },
+  checkinMargin: 2, // In minutes. Optional.
+  maxRuntime: 10, // In minutes. Optional.
+  timezone: "America/Los_Angeles", // Optional.
+};
+
+// 🟡 Notify Sentry your job is running:
+const checkInId = Sentry.captureCheckIn(
+  {
+    monitorSlug: "<monitor-slug>",
+    status: "in_progress",
+  },
+  monitorConfig
+);
+
+// Execute your scheduled task here...
+
+// 🟢 Notify Sentry your job has completed successfully:
+Sentry.captureCheckIn(
+  {
+    // Make sure this variable is named `checkInId`
+    checkInId,
+    monitorSlug: "<monitor-slug>",
+    status: "ok",
+  },
+  monitorConfig
+);
+```
+
+Available monitor config properties:
+
+`schedule`:
+
+: The job's schedule :
+
+The schedule representation for your monitor, either `crontab` or `interval`. The structure will vary depending on the type:
+
+```json
+{"type": "crontab", "value": "0 * * * *"}
+{"type": "interval", "value": "2", "unit": "hour"}
+```
```

Comment:
```suggestion
  The schedule representation for your monitor, either `crontab` or `interval`. The structure will vary depending on the type:
  
  ```json
  {"type": "crontab", "value": "0 * * * *"}
  {"type": "interval", "value": "2", "unit": "hour"}
  ```
```

Small nit, but can we tab these in a bit so the info organization is visually clearer?